build_docker.sh

```diff
@@ -1 +0,0 @@
-docker build -t mottosso/qt.py27 -f Dockerfile-py2.7 .
```

Comment:
Yeah, sure. It's just one command per dockerfile now: `docker build . -f Dockerfile.vfxplatform2017 -t fredrikaverpil/qt.py:2017`

Are you sure you don't want that in `DOCKER.md` instead though?